spec/rails_helper.rb

```diff
@@ -55,6 +55,12 @@
       VendorValidatorJob.perform_now(*args)
     end
   end
+
+  config.around(:each, user_flow: true) do |example|
+    Capybara.current_driver = :rack_test
```

Comment:
this is the opposite of our `js: true` -- don't some of our user flow tests use JS?

Reply:
The default driver is `rack_test_desktop` which does not support screenshots, so it isn't quite the opposite of `js: true`.

Also, there's issues with poltergeist coming from the SP so our JS user flows are disabled right now.

Reply:
Ok thanks for clarifying!